README.md

```diff
@@ -219,40 +219,40 @@ Use this command to generate self-signed TLS certificate:
 - `GET /api/files`
 
   **Params:**
-  - **expr**: logical expression. Example: `!(12&15)&(12|15)` means all files with single tag with id `12` or `15`
-  - **search**: text (or regexp) for search
-  - **regexp**: is search a regular expression (it is true when regexp != "")
-  - **sort**: name | size | time
+  - **expr**: logical expression. Example: `!(12&15)&(12|15)` means all files that have single tag with the id `12` or `15`
+  - **search**: a text/regexp search query
+  - **regexp**: allow regexp search queries (`true` when `regexp != ""`)
```

Comment:
It can be replaced with "enable regexp search (it is `true` when param is not an empty string)"